docs/en/ingest-management/release-notes/release-notes-8.10.asciidoc

```diff
@@ -323,6 +323,51 @@ _Example 3: Ansible_
 
 ====
 
+[[known-issue-166553]]
+.Filtering Elastic Agents in Kibana generates an "Error fetching agents" message
+[%collapsible]
+====
+
+*Details*
+
+A {kibana-ref}/kuery-query.html[KQL query] in a Fleet search field now returns a `400` error when the query is not valid.
+
+Previously, the search fields would accept any type of query, but with the merge of {kibana-pull}161064[#161064] any type of KQL sent to {fleet} needs to have a valid field name, otherwise it returns an error.
+
+Note that this is expected behaviour.
```

Comment:
I'm just not sure if we should keep this statement here, as many users are seeing this as a regression and we are going to find a solution for it. What do you think to remove it?

Reply:
Good point! I've removed the sentence (via [88e443d](https://github.com/elastic/ingest-docs/pull/523/commits/88e443d78fd959d9c9b2a8cacf6750d6c1177c73)).